packages/smoldot-test-utils/src/index.ts

```diff
@@ -164,23 +168,24 @@ export const respondWith = (jsonResponses: string[]) => {
  * first returning the subscription id.  The second a response to the
  * subscription.
  */
-const createRequestProcessor = (options: SmoldotOptions, responder: RpcResponder, healthResponder: RpcResponder) => {
-  return (rpcRequest: string, chainIndex: number) => {
+const createRequestProcessor = (options: SmoldotAddChainOptions, responder: RpcResponder, healthResponder: RpcResponder) => {
+  return (rpcRequest: string) => {
+    // eslint-disable-next-line @typescript-eslint/no-unsafe-call
```

Comment:
We should review all the lint disable statements when we remove the temporary mock for < 0.3.x

I'm suspicious of the need for this one too.